include/katanoisi/classifier.h

```diff
@@ -14,7 +14,7 @@ class classifier {
   }
 
   std::vector<std::pair<fasttext::real, std::string>>
-  prediction(std::__cxx11::string &text, int count);
+  prediction(std::__cxx11::string &text, int count, float threshold=0.0);
```

Comment:
Why a `float` not a `double`?

Reply:
Changed in next commit

Reply:
Actually, FastText expects a `real` value for the threshold parameter, that they defined as a `float`.